.github/workflows/code-quality.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          npm install
+          npm install --legacy-peer-deps
```

Comment:
I've added this line because sometimes the build seems to fail, maybe the runners are migrating to node. I preferably stick with a more recent (latest) node on the ci so we catch stuff early. Figuring out why/if we need legacy-peer-deps is IMO another task.